treeherder/webapp/api/performance_data.py

```diff
@@ -36,16 +37,16 @@ def get_performance_series_summary(self, request, project, jm, pk=None):
         """
         GET method implementation for listing signatures
 
-        Input: time interval
+        Input: time interval, machine_platform
         Output: all series signatures and their properties
         """
         try:
-            interval = int(request.QUERY_PARAMS.get('interval'))
+            interval_second = int(request.QUERY_PARAMS.get('interval'))
+            machine_platform = request.QUERY_PARAMS.get('machine_platform')
```

Comment:
interval_second, if you were going to change it, should be interval_seconds (since its a plural)

However, no need to change it since the shortform is better. Also I'd change machine_platform to just platform right here, for brevity. So:

    interval_seconds = int(request.QUERY_PARAMS.get('interval'))
    platform = request.QUERY_PARAMS.get('machine_platform')

Then pass them to get_performance_series_summary below.